### PR TITLE
added repos/list_contributors

### PR DESCRIPTION
**Pull request by dmgorsky**
resolving #490

### CODE REVIEW

**Comment by dmgorsky** [comment]
@XAMPPRocky

File: src/api/repos/contributors.rs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        Self {
+            handler,
+            anon: None,
+            per_page: None,
+            page: None,
+        }
+    }
+
+    /// Set to 1 or true to include anonymous contributors in results.
+    pub fn anon(mut self, include_anon: impl Into<bool>) -> Self {
```
Comment (manchicken) [inline src/api/repos/contributors.rs:26]: Dumb question: does it matter that these aren't `&mut self`?

**Review by manchicken — COMMENTED** [review]
I think it would be nice if you added a test as well.

File: src/api/repos/contributors.rs (hunk 1 above)
Comment (dmgorsky) [reply src/api/repos/contributors.rs:26]: Here we consume `self` and then return it, as a part of a builder pattern.

Comment (manchicken) [reply src/api/repos/contributors.rs:26]: That makes sense, sorry I missed it the first go-round.

**Comment by manchicken** [comment]
This looks great, I can't wait to use it!

**Comment by dmgorsky** [comment]
@XAMPPRocky This workflow requires approval from a maintainer 😇

**Comment by XAMPPRocky** [comment]
Thank you for your PR, and congrats on your first contribution! 🎉

**Pull request merged**